src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/MediaPlayerElement/MediaPlayerElement_ManualTest.xaml.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement;
+
+[SampleControlInfo("MediaPlayerElement", "ManualTest", ignoreInSnapshotTests: true, description: "To make everybody happy, here is the Big Buck Bunny sample", IsManualTest = true)]
```

Comment:
> To make everybody happy, here is the Big Buck Bunny sample

that is a good description of the page...
but you should put the textblocks content here
(pick one: either keep description or the textblock)

Reply:
I kept the textblock because it is possible to include more explanation. I removed the description.